rules/windows/other/win_wmi_persistence.yml

```diff
@@ -1,33 +1,52 @@
+action: global
 title: WMI Persistence
 id: 0b7889b4-5577-4521-a60a-3376ee7f9f7b
 status: experimental
-description: Detects suspicious WMI event filter and command line event consumer based on event id 5861 and 5859 (Windows 10, 2012 and higher)
-author: Florian Roth
+description: Detects suspicious WMI event filter and command line event consumer based on WMI and Sysmon logs.
+author: Florian Roth, Gleb Sukhodolskiy, Timur Zinniatullin oscd.community
 date: 2017/08/22
-modified: 2020/08/23
+modified: 2020/10/13
 references:
     - https://twitter.com/mattifestation/status/899646620148539397
     - https://www.eideon.com/2018-03-02-THL03-WMIBackdoors/
+    - https://attack.mitre.org/techniques/T1546/003/
 tags:
     - attack.persistence
     - attack.privilege_escalation
     - attack.t1084           # an old one
     - attack.t1546.003
+falsepositives:
+    - Unknown (data set is too small; further testing needed)
+level: medium
+---
 logsource:
     product: windows
-    service: wmi
+    service: wmi #native windows detection
+    definition: 'WMI Namespaces Auditing and SACL should be configured, EventID 5861 and 5859 detection requires Windows 10, 2012 and higher'
 detection:
-    selection:
+    wmi_filter_to_consumer_binding:
         EventID: 5861
     keywords:
         Message:
```

Comment:
Let's use field modifier `contains` here. We try to use field modifiers wherever possible. 😃